### PR TITLE
Unified UX for local and remote evaluations

### PR DESCRIPTION
**Pull request by dmpiergiacomo**
This PR relates to https://github.com/lynxius/lynxius-docs/pull/36.

In this PR:

* Both local and remote evaluations are now called with the same method `client.evaluate()`
* Local or remote evaluations are determined by the `run_local` parameter used to instantiate `LynxiusClient`
* Split `GettingStarted.ipynb` in `eval_locally.ipynb` and `eval_remotely.ipynb`
* Refactored `AI_medical_scribe.ipynb` to use new APIs
* Minor other fixes

### CODE REVIEW

**Comment by dmpiergiacomo** [comment]
Closing this as I want to unify the interface even more and I need more time. Will open a new PR later today.

**Comment by dmpiergiacomo** [comment]
Closing this as I want to unify the interface even more and I need more time. Will open a new PR later today.

File: lynxius/client.py [hunk 1]
```diff
@@ -60,31 +63,9 @@ def evaluate(self, eval: Evaluator) -> str | None:
...
-        Returns an eval run ID.
-        """
-
-        if not isinstance(eval, EvaluatorLocal):
-            raise ValueError(
-                "You are trying to store an evaluator that is not locally executed. "
-                "Please use a corresponding local evaluator class."
-            )
+        # Local evaluation
+        if self.run_local:
```
Comment (andrius-k) [inline lynxius/client.py:67]: Does the client need `run_local` flag? What about removing it from the client and doing this instead:
``` python
if eval.run_local:
  eval.evaluate_local()
```
Then we can use the same client for both types of evals.

Comment (dmpiergiacomo) [reply lynxius/client.py:67]: Well spotted! I removed `run_local` from all evals and kept it in the client. Keeping this parameter on the client translates into changing only one line in the code for users if they want to switch from evaluations run locally to evaluations run remotely.

**Review by andrius-k — APPROVED** [review]

**Pull request merged**